src/main/java/Duke.java

```diff
@@ -1,10 +1,121 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    public enum commands {
+        BYE, LIST, MARK, UNMARK, TODO, DEADLINE, EVENT, DELETE;
+
+        @Override
+        public String toString() {
+            return name().toLowerCase();
+        }
+    }
+
+    private static String numOfTasks(ArrayList<Task> tasks) {
+        return tasks.size() == 1 ? " task " : " tasks ";
+    }
+
     public static void main(String[] args) {
+        ArrayList<Task> tasks = new ArrayList<>();
+        Scanner sc = new Scanner(System.in );
```

Comment:
Should there be no extra space after `System.in`?
```suggestion
        Scanner sc = new Scanner(System.in);
```